src/components/BookForm.js

```diff
@@ -0,0 +1,27 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+
+const BookForm = ({ addBook }) => {
+  const inputRef = useRef();
+
+  const submitBookHandler = (e) => {
+    e.preventDefault();
+    addBook(inputRef.current.value);
+    inputRef.current.value = '';
+  };
+
+  return (
+    <div>
+      <form onSubmit={submitBookHandler}>
+        <input ref={inputRef} />
+        <button type="submit">Add</button>
+      </form>
```

Comment:
- [ ] It would be better if we have inputs for both `title` and `author` since creating or adding a book requires those inputs

Reply:
- Well resolved 👍🏽